app.py

```diff
@@ -6,15 +6,20 @@
 import redis
 
 from flask import Flask, jsonify, send_from_directory
+from flask_cors import CORS
 app = Flask(__name__, static_url_path='', static_folder='static')
+CORS(app)
 main_redis = redis.Redis(decode_responses=True, db=0)
 stats_redis = redis.Redis(decode_responses=True, db=1)
 
 
-def getStreams(count = 1):
+def getStreams(count = 1, trainId=None):
```

Comment:
What about breaking this out to keep `getStreams` pretty simple and have a separate for getting the raid stream?

Reply:
Yeah I think a single `getRaid` as you mentioned in the comment below will do well for us here. We can get rid of the `getTrainId` and this modification with this change.